keyboards/dz60/rules.mk

```diff
@@ -55,4 +55,4 @@ BACKLIGHT_ENABLE = yes  # Enable keyboard backlight functionality
 AUDIO_ENABLE = no
 RGBLIGHT_ENABLE = yes
 
-LAYOUTS = 60_ansi 60_iso 60_hhkb
+LAYOUTS = 60_ansi 60_ansi_split_bs_rshift 60_hhkb 60_iso
```

Comment:
The general convention so far, has been to include a keymap within `layouts/default/60_ansi_split_bs_rshift` when adding a new community layout. I think it is to lock in the expectations around LAYOUT macro name and order.

Reply:
The default keymap is already checked in: https://github.com/qmk/qmk_firmware/blob/master/layouts/default/60_ansi_split_bs_rshift/default_60_ansi_split_bs_rshift/keymap.c

Sorry, to add context, I didn't create this community layout, so I don't know the history. It's already checked-in and in use. I just wanted to add support for it to a new PCB so I can use it too.

As for which other keyboards use it, I did a quick search in the repro and it looks like Clueboard, Infinity60, GH60, D060, and a couple others already declare support for 60_ansi_split_bs_rshift layout in rules.mk.

Reply:
Somehow completely missed it, ignore my comment completely.